components/tikv_alloc/Cargo.toml

```diff
@@ -16,6 +16,9 @@ mem-profiling = ["jemallocator/profiling"]
 [dependencies]
 libc = "0.2"
 log = "0.3"
+lazy_static = "0.2.1"
```

Comment:
IMO, I don't want this component depends on future crates.

Reply:
`FutureMutex` is needed for running two requests without blocking whole thread. Without this I think I can only return an Error to user while another request is handling (with an atomic variable to record whether a profiler is running).

Reply:
As jay suggestion, I also prefer removing this to TiKV itself, not in this crate.

It is strange to depend on the futures crate.